base/util/src/main/java/com/netscape/cmsutil/crypto/CryptoUtil.java

```diff
@@ -2568,6 +2604,50 @@ public static List<byte[]> exportSharedSecret(String nickname, java.security.cer
         return listWrappedKeys;
     }
 
+    public static List<byte[]> exportSharedSecret(String nickname, java.security.cert.X509Certificate wrappingCert,
```

Comment:
I know this function (exportSharedSecret) is placed right after the original exportSharedSecret which assumed DES3 sym wrapping key.  I suggest you give this function its own comment in case they got separated later, such as 
/*
 * Return a list of two wrapped keys:
 * first element: temp AES key wrapped by cert,
 * second element: shared secret wrapped by temp AES key
 * /

Reply:
Taken care of.